components/search-tab-group/search-tab-group.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```

Comment:
Tab switching logic and functionality are pretty much the same as what we already have in `profile-tab-group.jsx`. I wanted this to be its own separate component as `profile-tab-group.jsx`'s logic is a lot more complex.